packages/vike-node/README.md

```diff
@@ -247,14 +237,46 @@ startServer()
 
 function startServer() {
   const app = new Elysia()
-  app.use(vike())
+  app.get('/*', vike())
   const port = +(process.env.PORT || 3000)
   app.listen(port, () => console.log(`Server running at http://localhost:${port}`))
 }
 ```
 
 ## Migration guide:
 
+### `0.1.x` to `0.2.x`
```

Comment:
How do we handle that? When merging we use a specific syntax to add breaking changes?
Should I just remove this migration from the `README` then, and put it here in a comment until we squash merge?

Reply:
When we squash merge we preserve all `BREAKING CHANGE:` commit messages. So that the squash commit contains a bunch of `BREAKING CHANGE:` lines which are picked by release-me which then updates CHANGELOG.md accordingly. So, yea, let's remove it from the readme.